ledger/acctdeltas.go

```diff
@@ -488,7 +485,6 @@ func (a *compactAccountDeltas) accountsLoadOld(tx *sql.Tx) (err error) {
 		case sql.ErrNoRows:
 			// we don't have that account, just return an empty record.
 			a.updateOld(idx, store.PersistedAccountData{Addr: addr})
-			err = nil
```

Comment:
removing this would break compacting functionality when one creates a new account: last account is a new, and accountsLoadOld would return an error. I guess we need a test for this.

Reply:
That was the linter giving an error and asking to remove it.

Reply:
weird, it is not shadowed, maybe having `return err` at the end would fix it

Reply:
ops, it is shadowed here
```
		rowid, acctDataBuf, err := arw.LookupAccountDataByAddress(addr)
```
and `TestAccountDBInMemoryAcct` is supposed to fix catch it